lib/tasks/create_tables.rake

```diff
@@ -0,0 +1,75 @@
+namespace :db do
+  desc 'Create all report schema tables'
+  task :create_tables do
+    include Report::MysqlConnection
+
+    create_hits_by_day reports_client
+    create_orders_by_day reports_client
+    create_import_logs reports_client
+  end
+end
+
+def create_hits_by_day(client)
+    query =<<EOF
```

Comment:
I'd rather keep this query into a separated file under /sql folder. It is easy to maintain and change. You do not have to do it now. But keep it in mind for a future refactor.